SecurityPkg/Tcg/Tcg2AcpiFfa/Tcg2AcpiFfa.inf

```diff
@@ -0,0 +1,77 @@
+## @file

```

Comment:
Can this driver be upstreamed to edk2?

Reply:
That is the idea. There is one missing functionality about physical presence interface before we can upstream. But I would not want to let that one piece to hold off the check in of this driver as some platforms needs it to boot to Windows.

Reply:
I don’t think it needs to wait, just wanted to make sure it was on the radar and if not to understand why